firmware-management/mbl-watchdog-init/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(mblWatchdogInit)
+
+set(MBL_WATCHDOG_INIT_SRC "${CMAKE_CURRENT_SOURCE_DIR}/mbl-watchdog-init.c")
+
+add_executable(mbl-watchdog-init ${MBL_WATCHDOG_INIT_SRC})
+
+target_compile_options(mbl-watchdog-init 
+    PUBLIC -Wall -Wpedantic -Werror
```

Comment:
I suggest making more things errors. `-Wall` leaves a lot of useful things out.

Reply:
131dc3fca138422be97e2ca07dbcca09f5c2cd14